test/integration/test_serialization.py

```diff
@@ -70,15 +77,23 @@ def create_cloudformation_stack_and_serialize(tmp_location_key_manager: Path, tm
         raise e
 
 
-def test_cloudformation_stack_with_local_stack(tmp_path, local_stack, default_asset_id, test_dummy_ami_id):
+def test_cloudformation_stack_with_local_stack(tmp_path, local_stack_aws_access, default_asset_id, test_dummy_ami_id):
     """
     Test that serialization and deserialization of CloudformationStack work!
     """
     tmp_file_key_file = Path(tmp_path) / "key_file_manager.data"
     tmp_file_cloud_formation = Path(tmp_path) / "cloudformation.data"
     q = mp.Queue()
-    p = mp.Process(target=create_cloudformation_stack_and_serialize,
-                   args=(tmp_file_key_file, tmp_file_cloud_formation, q, default_asset_id, test_dummy_ami_id))
+    p = mp.Process(
+        target=create_cloudformation_stack_and_serialize,
+        args=(
+            local_stack_aws_access,
```

Comment:
Forwarding local_stack_aws_access to the process is, in my option, not the best idea, because it might get serialized.

Reply:
Thank you. I had similar thoughts but picket the easy way first.
See next push for update.